app/src/main/java/com/rockthevote/grommet/ui/registration/personal/AdditionalInfoFragment.java

```diff
@@ -191,6 +191,7 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
             phoneTypeSpinner.getEditText().setText(phoneTypeEnumAdapter.getItem(0).toString());
         }
 
+        // TODO ensure partnerNamePref and partnerVolunteerText gets updated again with partner response
```

Comment:
This is just a reminder so we can go back and readd these at the end if we incidentally didn't. As is, these are empty which is causing one checkbox to be empty and the other to have an empty format string.

Reply:
we're going to remove the shared preferences model in favor of using the database, I couldn't do it yet because it was causing too many cascading bugs. Still a good comment though.